### PR TITLE
Support Accumulo 2.1

### PR DESCRIPTION
**Pull request by cfkoehler**

### CODE REVIEW

**Review by apmoriarty — DISMISSED** [review]
LGTM

**Comment by MiguelRicardos** [comment]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -555,10 +586,6 @@
       <groupId>gov.nsa.datawave.microservice</groupId>
       <artifactId>spring-boot-starter-datawave</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.apache.accumulo</groupId>
-      <artifactId>accumulo-core</artifactId>
-    </dependency>
```
Comment (jpdahlke) [inline pom.xml:561]: We should probably explicitly manage this dependency

**Review by jpdahlke — APPROVED** [review]

**Comment by MiguelRicardos** [comment]
Looks good

**Review by MiguelRicardos — APPROVED** [review]

**Pull request merged**